uw-frame-components/portal/widgets/partials/type__option-link.html

```diff
@@ -2,7 +2,7 @@
 
   <!-- OPTION-LINK ICON -->
   <div class="option-link-icon">
-    <a href="{{ widget.selectedUrl }}" target="_self" rel="noopener noreferrer">
+    <a href="{{ widget.selectedUrl }}" target="{{ widget.target ? widget.target : '_self' }}" rel="noopener noreferrer">
```

Comment:
I'm starting to be sold on the idea of auto-determining target depending upon the value of the `href`. As in, if it starts with `/`, it's `_self`; if it starts with `http`, it's `_blank` with the `rel="noopener noreferrer"`.